GitCommands/RevisionReader.cs

```diff
@@ -181,7 +181,7 @@ public sealed class RevisionReader : IDisposable
                     new ArgumentBuilder
                     {
                         { refFilterOptions.HasFlag(RefFilterOptions.Reflogs), "--reflog" },
-                        { AppSettings.SortByAuthorDate, "--author-date-order" },
+                        { AppSettings.SortByAuthorDate, "--author-date-order", "--topo-order" },
```

Comment:
`--topo-order` is a filter too, this cannot be done this way.
Branches not merged to HEAD are not shown.

So this need to be displayed with filters too.

Reply:
> Branches not merged to HEAD are not shown.

Sure? I would have expected an explicit warning in the git documentation. At least, I have not noticed it.

![image](https://user-images.githubusercontent.com/36601201/144320072-8430f78f-0fc9-489a-bd7f-81ceef2ff53d.png)

Reply:
Inserting in branches, could it be different with other options?
(I do not see your commit with this option)
Git 2.34.1

`-c log.showSignature=false log --max-count=100000 -z --topo-order --pretty=format:"%H%T%P%n%at%n%ct%n%aN%n%aE%n%cN%n%cE%n%B"`

I interpret the following as a filter:
```
Show no parents before all of its children are shown, and avoid showing commits on multiple lines of history intermixed.
```

Reply:
If added using your tweak, `--all` is omitted. With this PR it is:
`-c log.showSignature=false log --max-count=100000 -z --pretty=format:"%H%T%P%n%at%n%ct%n%aN%n%aE%n%cN%n%cE%n%B" --topo-order --all --boundary --not --glob=notes --not`
So in combination with filters, we must double-check if `--topo-order` causes side-effects.

Reply:
`--all` is added so it seem OK.

Then it likely comes down to implementing the dropdown (I prefer the current way to display).